README.md

```diff
@@ -16,3 +18,18 @@ What, no tests?! This is sacrilege!
 
 Resourceful allows us to declaratively define an API with almost no code. Until
 there is need for actual code, we will put off setting up a test framework.
+
+Deployment
+----------
+
+Continuous deployment is used for this application. Continuous deployment aims to automate the process of deploying code. Each stage in the process is automated, so that every change that passes all the tests is deployed to production automatically. These are the steps in the process:
```

Comment:
@jdesrosiers, changes have been made. Please let me know what other comments you have.